pages/series/[slug].vue

```diff
@@ -63,6 +71,86 @@ const parsedCarouselData = computed(() => {
     }
   })
 })
+
+// Transform Data for Tabbed Section
+const parsedUpcomingEvents = computed(() => {
+  // fail gracefully if data does not exist (server-side)
+  if (!upcomingEvents.value)
+    return []
+
+  // Transform data
+  return upcomingEvents.value.map((item, index) => {
+    return {
+      ...item,
+      image: item.image[0]
```

Comment:
Make it fail safe: 
`image: item.image && item.image.length > 0 ? item.image[0] : null`

Reply:
✅